### PR TITLE
Docker only with no garmin upload

### PR DESCRIPTION
**Pull request by jyc66**

### CODE REVIEW

**Comment by jyc66** [comment]
I modified the code to be more transparent to non-docker users by adding environment variables that are only used for docker.

File: peloton-to-garmin.py [hunk 1]
```diff
@@ -96,4 +108,3 @@
 
 logger.info("Done!")
 logger.info("Your Garmin TCX files can be found in the Output directory: " + output_directory)
-os.system("pause")
```
Comment (jyc66) [inline peloton-to-garmin.py]: I had to remove this to ensure the container would run and exit automatically

Comment (philosowaffle) [reply peloton-to-garmin.py]: Let's make this configurable. Non-docker users may prefer to still have the output window to see any errors that were written to the console.  Maybe we can put it behind a `DebugMode` flag in the config.ini?  Default could be `false`, i.e. don't pause the console.

Comment (jyc66) [reply peloton-to-garmin.py]: Alright, I added a flag and also changed the pause command to something that works on a mac and hopefully is platform agnostic.

**Pull request merged**